rllib/utils/compression.py

```diff
@@ -33,7 +33,7 @@ def pack(data):
         data = lz4.frame.compress(data)
         # TODO(ekl) we shouldn't need to base64 encode this data, but this
```

Comment:
can you get rid of this comment too if this is working now.

Reply:
I believe the comment is actually still relevant—theoretically, you should be able to store the raw bytes output of `lz4.frame.compress`. I'm not sure why base64 encoding is necessary and I didn't want to remove it entirely, so the only change I did is to store the base64 encoded version as bytes instead of unicode. If we removed the base64 encoding it would reduce memory consumption by another 25% but I didn't want to mess with it due to this comment.

Reply:
oh I see. get it. that's cool.
can you just rebase and trigger CI one more time please?
thanks.